RevenueCatUI/View+PresentPaywall.swift

```diff
@@ -38,13 +47,15 @@ extension View {
     /// If `nil` (the default), `Offerings.current` will be used. Note that specifying this parameter means
     /// that it will ignore the offering configured in an active experiment.
     /// - Parameter fonts: An optional `PaywallFontProvider`.
+    /// - Parameter presentationMode: The desired presentation mode of the ``PaywallView``. Defaults to ``.sheet``.
     ///
     /// ### Related Articles
     /// [Documentation](https://rev.cat/paywalls)
     public func presentPaywallIfNeeded(
         requiredEntitlementIdentifier: String,
         offering: Offering? = nil,
         fonts: PaywallFontProvider = DefaultPaywallFontProvider(),
+        presentationMode: PaywallPresentationMode = .sheet,
```

Comment:
Let's add this extension so that we don't repeat the default everywhere:
```swift
internal extension PaywallPresentationMode {

    static let `default`: Self = .sheet

}
```

Reply:
good point! i tend to go DRY (don't repeat yourself) when it's 3 o more